eclair-core/src/test/scala/fr/acinq/eclair/wire/TlvCodecsSpec.scala

```diff
@@ -308,6 +306,16 @@ class TlvCodecsSpec extends FunSuite {
     assert(lengthPrefixedTestTlvStreamCodec.encode(stream).require.toByteVector === hex"0f 01012a 0b012b 0d012a fd00fe02002a")
   }
 
+  test("encode/decode even generic tlv types above high range") {
```

Comment:
The spec calls those "custom records", I think we should align with that.

```suggestion
  test("encode/decode custom even tlv records") {
```

Reply:
Done in 73b82dd